senders/pushover/pushover.go

```diff
@@ -71,21 +75,34 @@ func (sender *Sender) makePushoverMessage(events moira.NotificationEvents, conta
 	return pushoverMessage
 }
 
-func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bool) string {
+func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bool, trigger moira.TriggerData) string {
 	var message bytes.Buffer
```

Comment:
Please, migrate this code to `strings.Builder`: https://medium.com/@felipedutratine/string-concatenation-in-golang-since-1-10-bytes-buffer-vs-strings-builder-2b3081848c45